src/main/java/org/springframework/data/couchbase/core/query/QueryCriteria.java

```diff
@@ -133,6 +247,26 @@ private String maybeWrapValue(Object value) {
 		}
 	}
 
+	private StringBuilder toQueryString(StringBuilder sb){
+		//StringBuilder sb=new StringBuilder();
+		String fieldName="`"+key+"`";
+		Object[] v=new Object[value.length+2];
+		v[0]=fieldName;
+		v[1]=operator;
+		for(int i=0; i< value.length;i++){
+			if(value[i] instanceof QueryCriteria)
+				v[i+2]="("+((QueryCriteria)value[i]).export()+")";
+			else
+				v[i+2]=maybeWrapValue(value[i]);
+		}
+
+		if(format == null)
```

Comment:
let's use { } always, even for single lines. It makes it much less error prone if refactoed later.